pkg/cluster/util.go

```diff
@@ -21,23 +21,47 @@ func isValidUsername(username string) bool {
 	return userRegexp.MatchString(username)
 }
 
+func isValidFlag(flag string) bool {
+	for _, validFlag := range []string{constants.RoleFlagSuperuser, constants.RoleFlagLogin, constants.RoleFlagCreateDB,
+		constants.RoleFlagInherit, constants.RoleFlagReplication, constants.RoleFlagByPassRLS} {
+		if flag == validFlag || flag == "NO"+validFlag {
+			return true
+		}
+	}
+	return false
+}
+
+func invertFlag(flag string) string {
+	if flag[:2] == "NO" {
+		return flag[2:]
+	} else {
+		return "NO" + flag
+	}
+}
+
 func normalizeUserFlags(userFlags []string) (flags []string, err error) {
```

Comment:
I think in this case there is no strong need for the named return values.
Besides, code like ``return nil, fmt.Errorf("bla")`` as for me looks more natural than 
```
err = fmt.Errorf("bla")
return
```